extensions/arc/src/ui/dashboards/postgres/postgresWorkerNodeParametersPage.ts

```diff
@@ -64,6 +64,17 @@ export class PostgresWorkerNodeParametersPage extends PostgresParametersPage {
 
 	protected refreshParametersTable(): void {
 		this._parameters = this._postgresModel.workerNodesEngineSettings.map(engineSetting => this.createParameterComponents(engineSetting));
-		this._parametersTable.data = this._parameters.map(p => [p.parameterName, p.valueContainer, p.description, p.resetButton]);
+
+		this._parametersTable.data = this._parameters.map(p => {
+			if (p.information) {
```

Comment:
This seems to be duplicated from the coordinator parameters page above - would suggest consolidating into one helper function.